cpp/src/parquet/properties.h

```diff
@@ -450,6 +451,22 @@ class PARQUET_EXPORT WriterProperties {
       return this->disable_statistics(path->ToDotString());
     }
 
+    /// Enable integer type to annotate decimal type as below:
```

Comment:
Is "Enable X to do Y" correct English? Just asking, as I'm not a native English speaker, but I would intuitively have expected "Allow X to do Y".